cmd/kube-apiserver/app/server.go

```diff
@@ -520,6 +522,10 @@ func buildGenericConfig(
 		lastErr = fmt.Errorf("failed to initialize admission: %v", err)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(genericfeatures.RequestManagement) {
```

Comment:
BTW, as I and others have remarked elsewhere, "RequestManagement" is a pretty vague term.  Since a feature name is user-facing and long-lived, we should use a better feature name.  Keep the bad old feature name as long as our retention policy requires, with the same meaning it has always had --- nothing.  But introduce a better new name.  Something like "APIPriorityAndFairness".